test/integration/functional_test.go

```diff
@@ -617,7 +617,16 @@ func validateCacheCmd(ctx context.Context, t *testing.T, profile string) {
 		t.Run("cache_reload", func(t *testing.T) { // deleting image inside minikube node manually and expecting reload to bring it back
 			img := "k8s.gcr.io/pause:latest"
 			// deleting image inside minikube node manually
-			rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", "sudo", "docker", "rmi", img))
+
+			var binary string
+			switch ContainerRuntime() {
+			case "docker":
+				binary = "docker"
+			case "containerd", "crio":
+				binary = "crictl"
```

Comment:
Can we use crictl for docker runtime too ?
( I think it's possible) if yes just .make everything use crictl

Reply:
no, deleting the image with crictl will not delete it from the docker daemon, so it can still be run with the docker runtime. we need to use the docker binary to correctly run this test.